flow/util/template_filters.py

```diff
@@ -128,11 +128,11 @@ def calc_memory(operations, parallel=False):
 
     Returns
     -------
-    float
+    int
         The reserved memory (numeric value) in gigabytes.
     """
     func = sum if parallel else max
-    return func(operation.directives["memory"] or 0 for operation in operations)
```

Comment:
I think this change is misleading and not consistent with our documentation that we support memory in `float`. Given that float values are not supported for great lakes, I think it'll be worth testing the recently introduced memory and wall time directives for all the environments. If a considerable amount of environments don't support float memory values (in GB) then we may want to default our directive to mega bytes.